packages/samples/headless-react/src/components/numeric-facet/numeric-facet.class.tsx

```diff
@@ -30,7 +30,7 @@ export class NumericFacet extends Component<NumericFacetProps> {
         field: props.field,
         facetId: props.facetId,
         generateAutomaticRanges: props.generateAutomaticRanges,
-        ...(props.currentValues && {currentValues: props.currentValues}),
+        currentValues: props.currentValues,
```

Comment:
This is a good example of where I encountered a bug due to the old way of handling `undefined`.

Reply:
Could the simpler fix be to specify an empty array? e.g.

```
currentValues: props.currentValues || []
```